frontend/src/modules/dashboard/views/ViewDropdown.tsx

```diff
@@ -1,3 +1,7 @@
+/**
+ * @summary Dropdown component Switch between different views
```

Comment:
Where would this dropdown component be used? Is this currently being used for anything, or can we deprecate it?

Reply:
Yes its not in use, am marking it as deprecated.